frontend/app/test/unit/setup.ts

```diff
@@ -3,6 +3,7 @@ import { cleanup, configure } from "@testing-library/react";
 import { afterEach, beforeAll, vi } from "vitest";
 import failOnConsole from "vitest-fail-on-console";
 
+import { resetDatabase } from "../../../lib/api-mocks/Database.ts";
```

Comment:
```suggestion
import { resetDatabase } from "@kiesraad/api-mocks";
```

Reply:
Good suggestion, simplified in cab555b3fa6c6d77069396d2b8abdeed52cd5475.